Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -32,3 +32,15 @@
     contents:
     - id: BoxSurvivalMedical
     - id: RubberStampPsychologist
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackCaptain
+  id: ClothingBackpackAdminAssistantFilled
```

Comment:
Null will tell you not to but I still believe this should be localised; cool RP role that could be used elsewhere :)

Reply:
Going to leave them unlocalised for now unless I see more interest in it being done.